### ISSUE TITLE
Our documentation does not adequately describe stewardship vs advisement

### ISSUE THREAD

**Issue by jessemapel** [planetarysoftware/TSC#50]
The docs are mostly written from the perspective of advisement. Documents such as the project lifecycle, application, and top level README need to be updated to account for both options.

**Comment by rbeyer** [comment]
I've just been looking over the documents and thinking about this, and I wondered whether we need to have these two different 'categories' of 'Top Level Project' at all?  Our thinking along the lines of 'who owns them' might be a false dichotomy (which may have been my fault).

We've been thinking about these two 'flavors' of project: (A) projects under advisement (which are projects that are full-fledged that 'just' want advice) and (B) projects that the TSC would become 'stewards' of (projects where their originators can't manage by themselves).  However, this distinction is just about how the 'project' arrives on our doorstep (and maybe what GitHub repo it ends up living in).

For flavor A, there is already a healthy group of individuals that want to form (or already de facto have) a Technical Committee (TC) for their project.  They may have existing infrastructure, sweat-equity, or the ability to bring resources to bear on the project.  All of which are great.  In this case, such a project would move through our incubation process, and then become a TSC-certified Planetary Software Top Level Project (TLP).  It is up to that TLP's TC to decide how much (or how little) interaction they have with the TSC, beyond what we minimally require (a representative from their TC must sit on our TSC).

For flavor B, there aren't five people (the minimum needed to form a TC) that have the available effort to 'run' the project and apply as a 'flavor A' project.  Here there are a few people that think that their project is important, and want to grow a community (either to participate in, or to walk away from, confident that the project will live on).  Here, they are coming to the TSC, and asking us to form a TC for the project.  The thing that occurred to me, is that as soon as we (the TSC) helps make a TC for a flavor B project, it suddenly becomes a flavor A project (because now it has a TC)!

So there aren't two kinds of TLPs, there are *only* TLPs.  Maybe we just offer two different approaches to enter incubation.  The 'normal' way is for a project is to form a TC and submit an application.  However, an individual could also request that the TSC consider a project, and if the TSC decided it was workable, the TSC would attempt to form a TC for that project, which would then submit an application via the 'normal' way.

TLPs could request a repo under our Planetary Software GitHub repo, or other services that the TSC might control or offer now or in the future, but they need not, that's up to their TC to decide.

It may certainly be my bias, because I just had this crazy idea, but this seems conceptually smoother to me, and we don't have to make an effort to differentiate between the different flavors of project, because after they have a TC, it doesn't matter how they got that way.

**Comment by rbeyer** [comment]
At our recent meeting, I think we agreed to approach this as I indicated above: i.e. there are *only* PlanetarySoftware Top Level Projects (not different kinds: there is no distinction between projects under 'advisement' and 'stewardship').  The only difference would be how a potential TLP might enter incubation (either with a full TC or with a partial TC, and they seek help from the TSC to grow their TC to the number of folks needed to get out of incubation), but once a project gets out of incubation, they're all just PlanetarySoftware TLPs.

This distinction still needs to be set down in the documentation we've talked about writing.
